manifests/post-installation/dpudeployment.yaml

```
@@ -0,0 +1,53 @@
+    ovn:
+      serviceTemplate: ovn
+      serviceConfiguration: ovn
+    doca-telemetry-service:
+      serviceTemplate: doca-telemetry-service
+      serviceConfiguration: doca-telemetry-service
+    doca-blueman-service:
+      serviceTemplate: doca-blueman-service
+      serviceConfiguration: doca-blueman-service
+  serviceChains:
+    switches:
+      - ports:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)